lib/utils/index.ts

```diff
@@ -3,3 +3,10 @@ export const isUndefined = value => typeof value === "undefined";
 export const isString = value => typeof value === "string";
 export const isNumber = value => typeof value === "number";
 export const isFunction = value => typeof value === "function";
+export const supportsCookies = () => {
+  try {
+    return !!navigator.cookieEnabled;
```

Comment:
not a big deal, but I'd make this
```suggestion
    return Boolean(navigator.cookieEnabled);
```